ns/init_linux.go

```diff
@@ -76,11 +76,28 @@ func getSupportedNlFamilies() []int {
 }
 
 func loadXfrmModules() error {
-	if out, err := exec.Command("modprobe", "-va", "xfrm_user").CombinedOutput(); err != nil {
-		return fmt.Errorf("Running modprobe xfrm_user failed with message: `%s`, error: %v", strings.TrimSpace(string(out)), err)
+	if err := loadModule("xfrm_user"); err != nil {
+		return err
 	}
-	if out, err := exec.Command("modprobe", "-va", "xfrm_algo").CombinedOutput(); err != nil {
-		return fmt.Errorf("Running modprobe xfrm_algo failed with message: `%s`, error: %v", strings.TrimSpace(string(out)), err)
+	if err := loadModule("xfrm_algo"); err != nil {
+		return err
 	}
 	return nil
 }
+
+func loadModule(module string) error {
+	if !isModuleAlive(module) {
```

Comment:
This is for avoiding to modprobe when we know from `/proc/modules` that the module is already there. Given we know there are cases when running inside a container where modprobe would fail for reasons not related to the bring up of the module.
This is expecially true when running docker daemon inside a container.